katportalclient/test/test_client.py

```diff
@@ -1331,6 +1333,50 @@ def test_modify_userlog(self):
             userlog_to_modify['tags'] = 'random nonsense'
             userlog = yield self._portal_client.modify_userlog(userlog_to_modify)
 
+    @gen_test
+    def test_sensor_subarray_lookup(self):
+        """Test sensor subarray lookup is correctly extracted."""
+        lookup_base_url = self._portal_client.sitemap['subarray'] + '/3/sensor-lookup/cbf/device_status/0'
```

Comment:
This line is a little bit too long.  Please keep <= 90 chars, as per our coding guidelines.
Same for the line in the other tests.

Reply:
Agreed. Lines like these you can format like:
```
lookup_base_url = '{}/3/sensor-lookup/cbf/device_status/0'.format(
    self._portal_client.sitemap['subarray'])
```
or even
```
lookup_base_url = (self._portal_client.sitemap['subarray'] +
                   '/3/sensor-lookup/cbf/device_status/0')
```

Reply:
I've addressed the suggested changes on the next commit